public/javascripts/components/conversation/conversation.js

```diff
@@ -103,6 +114,13 @@ window.ChubboChat.components.conversation = Vue.extend({
         me.messages.push(me.surveyQuestions[0]);
         me.surveyQuestions.splice(0, 1);
       }
+    },
+    sendToDatabase: function() {
+      window.ChubboChat.services.surveyApi.sendSurveyResponses(
```

Comment:
Is there any way we could pull this window.whatever out into an instance variable or a local variable or something? If you ever decided to upgrade this thing to ES6 or webpack or anything it'd be a nightmare to go around and find all the window.stuffs - buuuutt if they were all at the top of the file, could be easy